SS1-Default_Public_Segment.md

```diff
@@ -62,15 +56,21 @@ The `channels` object contains all the information needed to see which protocols
 
 * Each channel is given a name as its key in the `channels` object which points to an object that stores the channel metadata.
 
-* If there is no `protocol` field set then the channel name is used as the protocol.
+* `protocol` (optional) If undefined then the channel name is used as the protocol.
+
+* `version` (optional) Used to specify a specific protocol version for the channel.
+
+* `update` (optional) The tick the channel was last updated on. This can be used by recieving parties to avoid loading segments and parsing data when it is not needed.
 
-* If the `segments` field is defined it should contain an array of segment IDs. To retrieve the message for the channel each segment should be combined together in the order defined by this array. If this field is used the `data` field should not be used.
+* `segments` (optional) A list of segment IDs used to reconstruct the channel message. To retrieve the message for the channel each segment should be combined together in the order defined by this array. If this field is used the `data` field should not be used.
 
-* If the `data` field is defined it will contain the message for the channel. This data must be a string, as different protocols can have different methods for serialization. If this field is used the `segments` field should not be used.
+* `data` (optional) contains the message for the channel. This data must be a string, as different protocols can have different methods for serialization. If this field is used the `segments` field should not be used.
 
-* The `version` field may optionally be defined for each channel.
+* `compressed` (optional) If true then the message was compressed using [lzstring's](https://github.com/pieroxy/lz-string) `compressToUTF16` function (to decompress use `decompressFromUTF16`).
```

Comment:
I'm aware that lzstring is the most used compression used in screeps, but in order to be open for different compression algorithms it might be a good idea to allow strings to reference the compression algorithm.
I'm totally fine using `true` to use lzstring though.